### PR TITLE
Add Dockerfile for pipeline, modifications to pipeline.

### LINKED ISSUE #94: Containerise Pipeline, Upload to Cloud.
It is crucial that:

- [x] The pipeline files are modified to function as a Python lambda function
- [x] A Dockerfile is created
- [x] The image is tested locally
- [x] The image is uploaded and tested as a lambda function
- [x] Relevant files are added to the repository

As a engineer, I need to be able to schedule the execution of the pipeline on the cloud, so that data is continuously added to the database

### PR DESCRIPTION
**Pull request by e-lemma**
I have:

- Edited `pipeline.py` to function as a Python lambda function
- Created a Dockerfile for the Pipeline
- Removed `asyncio` from any requirement.txt files

Closes #94

### CODE REVIEW

File: pipeline/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,4 @@
 aiohttp
-asyncio
```
Comment (Zhi-704) [inline pipeline/requirements.txt:2]: Oops my bad, thanks a lot for removing this

Comment (alina-101) [inline pipeline/requirements.txt:2]: Good spot that this doesn't need to be in `requirements.txt`

File: pipeline/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+COPY extract.py .
+COPY transform.py .
+COPY load.py .
+COPY pipeline.py .
+
+CMD [ "pipeline.main" ]
```
Comment (Lasped13) [inline pipeline/Dockerfile:11]: Good work!

File: pipeline/test_pipeline.py
Comment (Lasped13) [inline pipeline/test_pipeline.py:1]: What is the test coverage for your file?

File: requirements.txt
Comment (Lasped13) [inline requirements.txt:1]: Why was asyncio removed?

File: pipeline/requirements.txt
Comment (Lasped13) [inline pipeline/requirements.txt:1]: Why is there multiple requirements.txt files?

Comment (e-lemma) [reply pipeline/requirements.txt:1]: asyncio is a standard python library, it doesn't need to be installed. Additionally, we ran into a time-consuming bug caused by a second version of asyncio being installed via pip.

File: requirements.txt
Comment (e-lemma) [reply requirements.txt:1]: asyncio is a standard python library, it doesn't need to be installed. Additionally, we ran into a time-consuming bug caused by a second version of asyncio being installed via pip.

File: pipeline/test_pipeline.py
Comment (e-lemma) [reply pipeline/test_pipeline.py:1]: One HUNDRED percent!!

File: pipeline/requirements.txt
Comment (e-lemma) [reply pipeline/requirements.txt:1]: Having multiple requirement.txt files for each part of the project allows for the containerised applications to install only what is required, avoiding potential bugs or bloat.

**Review by Lasped13 — APPROVED** [review]

**Pull request merged**